internal/controller/component_build_controller_pac.go

```diff
@@ -423,6 +423,15 @@ func generateMergeRequestSourceBranch(component *appstudiov1alpha1.Component) st
 	return fmt.Sprintf("%s%s", pacMergeRequestSourceBranchPrefix, component.Name)
 }
 
+// getPipelineRunDefinitionFilePath returns full path in git repository to the pipeline run definition of the given Component.
+func getPipelineRunDefinitionFilePath(component *appstudiov1alpha1.Component, isPullRequest bool) string {
```

Comment:
probably better in internal/controller/component_build_controller_pipeline.go

Reply:
just FYI this will work ONLY if they didn't rename pipeline run files (about which we can't do much)

Reply:
> probably better in internal/controller/component_build_controller_pipeline.go

By name - yes, but in fact it's not used there, so I'd prefer to keep it closer to the main point of usage.